cookiecutter.json

```
@@ -8,6 +8,7 @@
     "author_name": "CERN",
     "author_email": "info@{{ cookiecutter.project_site }}",
     "year": "{% now 'local', '%Y' %}",
+    "python_version": ["3.8"],
```

```suggestion
    "python_version": ["3.8", "3.9"],
```

I would also 3.9 for the "adventurous" :)

Note: I think I understand where the `.split()[0]` comes from now... We sometimes put a choice like "3.9 (experimental)", which would then need to be parsed.

---

Added option `3.9` to `cookiecutter`